packages/server/src/orders/database/OrderSchema.ts

```diff
@@ -0,0 +1,74 @@
+import { Prop, raw, Schema, SchemaFactory } from '@nestjs/mongoose';
+import mongoose, { Document } from 'mongoose';
+
+import { Address } from '../../addresses/database/AddressSchema';
+import { User } from '../../users/database/UserSchema';
+
+export type OrderDocument = Order & Document;
+
+const ORDER_CONSTANTS = Object.freeze({
+  COMMENT: Object.freeze({
+    MAX_LENGTH: 300,
+  }),
+  PRICE: Object.freeze({
+    MIN: 0,
+  }),
+  COUNT: Object.freeze({
+    MIN: 1,
+  }),
+});
+
+class SubOrder {
+  @Prop({ required: true })
+  deliveryDate: Date;
+
+  @Prop({ required: true })
+  hourStart: Date;
+
+  @Prop({ required: true })
+  hourEnd: Date;
+
+  @Prop({ default: false })
+  delivered: boolean;
+}
```

Comment:
Chyba brakuje listy dań. Bo każde nasze podzamówienie (zamówienie na dany dzień) powinno się składać z daty i godziny dostarczenia + z listy dań składających się na to zamówienie.

Reply:
to w sumie ta sama moja wcześniejsza pomyłka w złym umieszczeniu `dishes`